export_job/export_zip_from_manifest_job.py

```diff
@@ -0,0 +1,184 @@
+import aiohttp
+import asyncio
+import json
+import os
+import shutil
+from gen3.tools.download.manifest import (
+    describe_access_to_files_in_workspace_manifest,
+    download_files_in_workspace_manifest,
+    list_files_in_workspace_manifest,
+)
+from gen3.auth import Gen3Auth
+import requests
+import boto3
+
+from temporary_api_key import TemporaryAPIKey
+
+
+CHUNK_SIZE = 10
+MANIFEST_FILENAME = "manifest.json"
+EXPORT_DIR = "export"
+OUTPUT_PREFIX = "[out] "
+MAX_DOWNLOAD_SIZE = 250000000
+DEFAULT_ERROR_MESSAGE = (
+    "Unable to complete this download. Please try again later, or use the Gen3 client."
+)
+
+
+async def build_manifest_from_study_ids(hostname, token, study_ids):
+    """
+    build a manifest from a list of metadata guids representing study ids
+    dumps result to gen3-sdk compatible manifest file
+    """
+    ongoing_requests = []
+    study_metadata = []
+
+    print(f"Assembling manifest for study ids: {study_ids}")
+
+    async with aiohttp.ClientSession() as client:
+        for study_id in study_ids:
+            ongoing_requests += [
+                client.get(
+                    f"https://{hostname}/mds/aggregate/metadata/guid/{study_id}",
+                    headers={"Authorization": f"bearer {token}"},
+                )
+            ]
+            if len(ongoing_requests) == CHUNK_SIZE:
+                study_metadata += await asyncio.gather(
+                    *[req.json() for req in await asyncio.gather(*ongoing_requests)]
+                )
+
+        study_metadata += await asyncio.gather(
+            *[req.json() for req in await asyncio.gather(*ongoing_requests)]
+        )
+
+    # print(f"Retrieved metadata: {study_metadata}")
+    manifest = []
+    for study in study_metadata:
+        if not study["__manifest"]:
+            print(
+                f"Study {study['project_number']} is missing __manifest entry. Skipping."
```

Comment:
Is the `key` configurable? Might it be `project_number` for some entries, but `study_id` for others?

Reply:
Might be worth removing this key altogether, since it's only ever needed for this informational log

Reply:
Yeah, it might be safer to not assume the content/structure of the data since the MDS really allows for any shape. I think you're right that `project_number` is most common. Maybe best to just set up that log line so it wouldn't error/crash if that field was missing